packages/cli/src/Doctor.ts

```diff
@@ -107,7 +107,7 @@ ${bold('Examples')}
       remoteModel: remoteModels[localModel.dbName ?? localModel.name],
     }))
 
-    const getFieldName = (f: DMMF.Field) => (f.dbNames && f.dbNames.length > 0 ? f.dbNames[0] : f.name)
+    const getFieldName = (f: DMMF.Field) => (f.dbName ?? f.name)
```

Comment:
```suggestion
    const getFieldName = (f: DMMF.Field) => f.dbName ?? f.name
```

Prettier removes the parens here, not sure why we don't catch this on CI.

Reply:
Made the tweaks in GH web since they were so small, and was indeed expecting CI to catch something like that. :)